tests/python/ci/test_ci.py

```diff
@@ -312,7 +314,7 @@ def test_cc_reviewers(
     """
     Test that reviewers are added from 'cc @someone' messages in PRs
     """
-    reviewers_script = REPO_ROOT / "ci" / "scripts" / "github_cc_reviewers.py"
+    reviewers_script = REPO_ROOT / "ci" / "scripts" / "github" / "github_cc_reviewers.py"
```

Comment:
do we think this pattern is common enough to make `GITHUB_SCRIPT_ROOT` and `JENKINS_SCRIPT_ROOT`?